commit abf7201316a357adf3ef64e70f1599c52edaf59e
Author: Peter Ujfalusi <peter.ujfalusi@ti.com>
Date:   Wed Nov 14 13:06:21 2018 +0200

    ASoC: omap-mcbsp: Fix latency value calculation for pm_qos
    
    [ Upstream commit dd2f52d8991af9fe0928d59ec502ba52be7bc38d ]
    
    The latency number is in usec for the pm_qos. Correct the calculation to
    give us the time in usec
    
    Signed-off-by: Peter Ujfalusi <peter.ujfalusi@ti.com>
    Acked-by: Jarkko Nikula <jarkko.nikula@bitmer.com>
    Signed-off-by: Mark Brown <broonie@kernel.org>
    Signed-off-by: Sasha Levin <sashal@kernel.org>

diff --git a/sound/soc/omap/omap-mcbsp.c b/sound/soc/omap/omap-mcbsp.c
index d0ebb6b9bfac..2d6decbfc99e 100644
--- a/sound/soc/omap/omap-mcbsp.c
+++ b/sound/soc/omap/omap-mcbsp.c
@@ -308,9 +308,9 @@ static int omap_mcbsp_dai_hw_params(struct snd_pcm_substream *substream,
 			pkt_size = channels;
 		}
 
-		latency = ((((buffer_size - pkt_size) / channels) * 1000)
-				 / (params->rate_num / params->rate_den));
-
+		latency = (buffer_size - pkt_size) / channels;
+		latency = latency * USEC_PER_SEC /
+			  (params->rate_num / params->rate_den);
 		mcbsp->latency[substream->stream] = latency;
 
 		omap_mcbsp_set_threshold(substream, pkt_size);
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1076_abf7201316a357adf3ef64e70f1599c52edaf59e.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1076_abf7201316a357adf3ef64e70f1599c52edaf59e.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
